tests/common/devices/multi_asic.py

```diff
@@ -594,3 +601,8 @@ def docker_copy_from_asic(self, container_name, src, dst, asic_id = 0):
         if duthost.is_multi_asic:
             container_name += str(asic_id)
         self.shell("sudo docker cp {}:{} {}".format(container_name, src, dst))
+        
+    def restart_bgp(self):
```

Comment:
why do we need to do `reset_service` every time ? Normally we need reset services only if services is not running properly

Reply:
This is refactored from original code, should I remove it?

Reply:
removed